crates/turborepo-lib/src/run/task_id.rs

```diff
@@ -14,6 +14,12 @@ pub struct TaskId<'a> {
     task: Cow<'a, str>,
 }
 
+impl Serialize for TaskId<'_> {
+    fn serialize<S: serde::Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        serializer.serialize_str(&self.to_string())
+    }
+}
```

Comment:
I think this is better expressed using [`#[serde(into = "String")]`](https://serde.rs/container-attrs.html#into)